client/payment-details/summary/index.tsx

```diff
@@ -51,6 +51,7 @@ import CancelAuthorizationButton from '../../components/cancel-authorization-but
 import { PaymentIntent } from '../../types/payment-intents';
 import DisputeAwaitingResponseDetails from '../dispute-details/dispute-awaiting-response-details';
 import DisputeResolutionFooter from '../dispute-details/dispute-resolution-footer';
+import ErrorBoundary from 'wcpay/components/error-boundary';
```

Comment:
Incredibly minor nitpick that is probably pedantic and not useful: remove `wcpay/` from `components/error-boundary` import to be consistent with other component imports within this file.

```suggestion
import ErrorBoundary from 'components/error-boundary';
```